arduino/libify.sh

```diff
@@ -118,20 +161,22 @@ function copy_dir() {
     pushd ${TARGET_LIB_DIR}/$1 >/dev/null
     
     if [ "x$VERBOSE" != "x" ]; then
-        echo "${OPENMRNPATH}/${2} ==> ${TARGET_LIB_DIR}/$1"
+        echo "${INVERSE_DIR}${OPENMRNPATH}/${2} ==> ${TARGET_LIB_DIR}/$1"
     fi
 
     if [[ "$OSTYPE" == "darwin"* ]]; then
-        cp -fa ${USE_LINK} ${OPENMRNPATH}/$2 .
+        cp -fa ${USE_LINK} ${INVERSE_DIR}${OPENMRNPATH}/$2 .
     else
-        cp -faxr ${USE_LINK} ${OPENMRNPATH}/$2 .
+        cp -faxr ${USE_LINK} ${INVERSE_DIR}${OPENMRNPATH}/$2 .
     fi
 
     popd >/dev/null
 }
 
-copy_file . arduino/{library.json,library.properties,keywords.txt,README.md,LICENSE,CONTRIBUTING.md}
-copy_dir . arduino/examples
+if [ "x$TARGET_IDF" == "x" ]; then
+    copy_file . arduino/{library.json,library.properties,keywords.txt,README.md,LICENSE,CONTRIBUTING.md}
+    copy_dir . arduino/examples
+fi
 
 copy_file src arduino/OpenMRNLite.{h,cpp} arduino/CDIXMLGenerator.hxx \
```

Comment:
Should we consider moving `arduino/OpenMRNLite.{h,cpp}` to the arduino block above since these have arduino dependencies?

Reply:
You have them in your repo. I guess you don't need it and never build it via cmake?

Reply:
I think they came over as a mass import from the exported tree create by the script. They are excluded from build via cmake: https://github.com/atanisoft/OpenMRNIDF/blob/5.1.0/CMakeLists.txt

Reply:
done